test/unit/services/messages/destroy_all_service_test.rb

```diff
@@ -12,28 +12,33 @@ def setup
   end
 
   def test_run!
-    assert_nil @message.deleted_at
+    Sidekiq::Testing.fake! do
 
-    Messages::DestroyAllService.run!({
-      account: @account,
-      association_class: MessageRecipient,
-      scope: :hidden
-    })
+      assert_nil @message.deleted_at
 
-    assert_not_equal nil, @message.reload.deleted_at
-  end
-
-  def test_run_with_sidekiq_job
-    assert @message.present?
-
-    perform_enqueued_jobs do
       Messages::DestroyAllService.run!({
         account: @account,
         association_class: MessageRecipient,
         scope: :hidden
       })
+      @message.reload
+
+      assert_not_equal nil, @message.reload.deleted_at
     end
+  end
 
-    assert_raise(ActiveRecord::RecordNotFound) { @message.reload }
+  def test_run_with_sidekiq_job
+    perform_enqueued_jobs do
+      Sidekiq::Testing.inline! do
```

Comment:
This is unnecessary now.
Please check this PR to understand why:
https://github.com/3scale/porta/pull/1640

Because this worker now inherits from ApplicationJob now.

And that PR affects this whole test, not just this line.